org.librarysimplified.r2.api/src/main/java/org/librarysimplified/r2/api/SR2Bookmark.kt

```diff
@@ -36,7 +37,19 @@ data class SR2Bookmark(
    * An estimate of the current progress through the entire book.
    */
 
-  val bookProgress: Double?
+  val bookProgress: Double?,
+
+  /**
+   * The URI of the bookmark, if any.
+   */
+
+  val uri: URI?,
```

Comment:
This changes the API in a way that is backward incompatible, so this should be a major version bump to 2.0.0-SNAPSHOT.